lib/ContainerManager.js

```diff
@@ -1087,6 +1087,18 @@ Server.prototype.streamFromDB = function(key) {
   return readStream;
 };
 
+/**
```

Comment:
New versions of Mocha won't exit if there are open processes. Such cases occur with open connection pools. This method is later call in the tests, so the server can be closed and Mocha can exit.